pandas/core/internals/managers.py

```diff
@@ -596,18 +596,24 @@ def replace_list(
         # figure out our mask apriori to avoid repeated replacements
         values = self.as_array()
 
-        def comp(s, regex=False):
+        def comp(s, regex=False, mask=None):
```

Comment:
are we *always* passing the mask? if so you can reorder the args and make it requires (as this is an internal function). If you can type the args would be great

Reply:
The mask is now required and the type of regex is set.